src/renderer/components/+catalog/internal-category-columns.tsx

```diff
@@ -119,3 +86,37 @@ export const defaultCategoryColumns: RegisteredAdditionalCategoryColumn[] = [
     searchFilter: entity => entity.status.phase,
   },
 ];
+
+const EntityName = observer(({ entity }: { entity: CatalogEntity }) => {
+  const hotbarStore = HotbarStore.getInstance();
```

Comment:
Usage of shared global state. Pass hotbar store as prop.

Reply:
Done